lib/prefill.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+module Declarations
+  module Prefills
+    class MonthlyPrefillService < ApplicationService
+      SENTRY_OWNER = Constants::SentryOwners::DECLARATIONS
+
+      Error = Exceptions::AppInternalError.with_scope('declarations.prefills.prefill_service')
+
+      class UnsupportedDeclaration < Error; end
+
+      private
+
+      def initialize(declaration:)
```

Comment:
Avoid using instance variables for the company and instead pass it as an argument to the `initialize` method. This will ensure that the service is explicitly initialized with the required context.